packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/signingout.md

```diff
@@ -1,94 +1,18 @@
-This singleton stores the token in user defaults. For more security, store the token in the Keychain.
+The next code snippet uses the `Credential` object of the `AuthFoundation` library to revoke the tokens for the current user. You can use other features of `Credential` to revoke the tokens of a specific user.
 
 ```swift
 
-import Foundation
 import OktaIdx
+import AuthFoundation
 
-enum TokenManagerError: Error {
-    case unableToReadToken(String)
-}
+class SignInController: InteractionCodeFlowDelegate {
+   ...
 
-class TokenManager {
-    static let shared = TokenManager()
+   // Sign out the current user.
+   public func signOut() throws {
+      //
+      try Credential.default?.remove()
```

Comment:
We want to avoid recommending `remove` since all that does is remove the tokens from the Keychain. Those tokens would still be valid though, so it could pose a potential risk if they had been intercepted.

I'd suggest instead using the async `revoke()` call.

Reply:
I'm converting all the code to async/await including this one. It will be in the next commit.